src/explore-education-statistics-frontend/src/modules/table-tool/TableToolPage-original.tsx

```diff
@@ -479,7 +479,11 @@ class TableToolPage extends Component<Props, State> {
                                   {permalinkLoading ? (
                                     <>
                                       Generating permanent link
-                                      <LoadingSpinner inline size={19} />
+                                      <LoadingSpinner
```

Comment:
This should be applied to the page TableToolPage.tsx (currently TableToolPage-original.tsx)